build/files.c

```diff
@@ -2659,24 +2659,285 @@ static int checkFiles(const char *buildRoot, StringBuf fileList)
     return rc;
 }
 
+static rpmTag copyTagsForDebug[] = {
+    /* see copyTagsDuringParse */
+    RPMTAG_EPOCH,
+    RPMTAG_VERSION,
+    RPMTAG_RELEASE,
+    RPMTAG_LICENSE,
+    RPMTAG_PACKAGER,
+    RPMTAG_DISTRIBUTION,
+    RPMTAG_DISTURL,
+    RPMTAG_VENDOR,
+    RPMTAG_ICON,
+    RPMTAG_URL,
+    RPMTAG_VCS,
+    RPMTAG_CHANGELOGTIME,
+    RPMTAG_CHANGELOGNAME,
+    RPMTAG_CHANGELOGTEXT,
+    RPMTAG_PREFIXES,
+    RPMTAG_DISTTAG,
+    RPMTAG_BUGURL,
+    0
+};
+
+static rpmTag copyTagsForDebugMain[] = {
+    RPMTAG_ARCH,
+    RPMTAG_SUMMARY,
+    RPMTAG_DESCRIPTION,
+    RPMTAG_GROUP,
+    /* see addTargets */
+    RPMTAG_OS,
+    RPMTAG_PLATFORM,
+    RPMTAG_OPTFLAGS,
+};
+
+/* this is a hack: patch the summary and the description to include
+ * the correct package name */
+static void patch_debug_package_string(Package dbg, rpmTag tag, Package pkg, Package mainpkg)
+{
```

Comment:
Just a stylistic nit, but this underscory function name looks a bit odd, surrounded by all those camelCases.